packages/terra-disclosure-manager/src/DisclosureManager.jsx

```diff
@@ -177,7 +177,7 @@ class DisclosureManager extends React.Component {
      * Allows a component to remove itself from the disclosure stack. If the component is the only element in the disclosure stack,
      * the disclosure is closed.
      */
-    delegate.dismiss = componentIndex > 0 ? popContent : this.safelyCloseDisclosure;
+    delegate.dismiss = popContent;
```

Comment:
Does this change need to be reflected in the goBack assignment as well?

Reply:
goBack is already using popContent (or nothing when we don't want to provide it)